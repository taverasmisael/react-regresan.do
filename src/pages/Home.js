import React, { PureComponent } from 'react'
import PropTypes from 'prop-types'
import Typography from 'material-ui/Typography'
export default class HomePage extends PureComponent {
  static propTypes = {
    prop: PropTypes.any
  }

  render() {
    return <Typography type="body1">Home</Typography>
  }
}
