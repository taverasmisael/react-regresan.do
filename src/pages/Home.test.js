import React from 'react'
import { shallow } from 'enzyme'
import Home from './Home'

describe('Pages/Home', () => {
  describe('rendering', () => {
    it('renders without crashing', () => {
      shallow(<Home />)
    })
    it('matches snapshot', () => {
      expect(shallow(<Home />)).toMatchSnapshot()
    })
  })
})
