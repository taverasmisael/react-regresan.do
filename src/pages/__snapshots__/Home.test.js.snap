// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pages/Home rendering matches snapshot 1`] = `
<React.Fragment>
  <WithStyles(Button)
    color="primary"
    onClick={[Function]}
    raised={true}
  >
    Login
  </WithStyles(Button)>
</React.Fragment>
`;
